mmdt/polls/views.py

```diff
@@ -3,20 +3,40 @@
 from django.urls import reverse
 from django.contrib import messages
 from .models import Question, Choice
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 
 
 class PollHomePage:
     def index(request):
-        latest_question_list = Question.objects.order_by('-pub_date')[:5]
+        # Get all polls, not just the latest 5
```

Comment:
remove this comment, the code and comment def is difference

Reply:
- [ ] I was fixed this comment.Thank you for your suggestion.